subprojects/core/src/main/java/org/gradle/testfixtures/internal/ProjectBuilderImpl.java

```diff
@@ -106,6 +106,8 @@ public ProjectInternal createProject(String name, File inputProjectDir, File gra
         File userHomeDir = gradleUserHomeDir == null ? new File(projectDir, "userHome") : FileUtils.canonicalize(gradleUserHomeDir);
         StartParameterInternal startParameter = new StartParameterInternal();
         startParameter.setGradleUserHomeDir(userHomeDir);
+        startParameter.useEmptySettings();
+        startParameter.setCurrentDir(inputProjectDir);
```

Comment:
Why do we need these two lines now? This affects production code folks use for testing.